packages/datadog-plugin-graphql/src/execute.js

```diff
@@ -7,18 +7,20 @@ let tools
 class GraphQLExecutePlugin extends TracingPlugin {
   static get id () { return 'graphql' }
   static get operation () { return 'execute' }
+  static get type () { return 'graphql' }
+  static get kind () { return 'server' }
 
   start ({ operation, args, docSource }) {
     const type = operation && operation.operation
     const name = operation && operation.name && operation.name.value
     const document = args.document
     const source = this.config.source && document && docSource
 
-    const span = this.startSpan('graphql.execute', {
+    const span = this.startSpan(this.operationName(), {
       service: this.config.service,
```

Comment:
While this does the job thanks to the default service computed when forwarding to the tracer's `startSpan` (see `TracingPlugin:startSpan`), this will not respond to changes in the naming schema's `serviceName()` function.

```suggestion
      service: this.config.service || this.serviceName(),
```